css/fractalCss/inputs.css

```
@@ -0,0 +1,30 @@
+/* Input Fields */
+
+.input {
+  width: 100%;
+  max-width: 70%;
```

```suggestion
  width: 70%;
```
Remove the above width property at line 4 and do changes in line as suggested. It will work the same.